implementations/rust/ockam/ockam_command/src/identity/default.rs

```diff
@@ -27,22 +26,30 @@ impl DefaultCommand {
 }
 
 fn run_impl(opts: CommandGlobalOpts, cmd: DefaultCommand) -> miette::Result<()> {
-    let state = opts.state.identities;
-    let idt = state.get(&cmd.name)?;
-    // If it exists, warn the user and exit
-    if state.is_default(idt.name())? {
-        Err(miette!(
-            "The identity '{}' is already the default",
-            &cmd.name
-        ))
+    if let Some(name) = cmd.name {
+        let state = opts.state.identities;
+        let idt = state.get(&name)?;
+        // If it exists, warn the user and exit
```

Comment:
I just noticed this should be changed to `// If it's already the default, warn the user and exit`